src/dots.js

```diff
@@ -59,10 +59,6 @@ export class Dots extends React.Component {
       );
     });
 
-    return (
-      <ul className={this.props.dotsClass} style={{display: 'block'}}>
-        {dots}
-      </ul>
-    );
+    return React.cloneElement(this.props.appendDots(dots), {className: this.props.dotsClass});
```

Comment:
@laveesingh Do we need cloneElement here ?

We can do some thing like this
```
const AppendDots = this.props.appendDots
return <AppendDots className={this.props.dotsClass} />
```

Reply:
I'll try and replace that.